graphics/diff-pdf/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                diff-pdf
+version             0.3
+categories          graphics
+maintainers         {@pietvo vanoostrum.org:piet}
+platforms           darwin
+supported_archs     i386 x86_64
+license             GPL-2
+
+description         diff-pdf is a tool for visually comparing two PDFs.
+long_description \
+    It takes two PDF files as arguments. By default, its only output is \
+    its return code, which is 0 if there are no differences and 1 if the \
+    two PDFs differ. If given the `--output-diff` option, it produces a \
+    PDF file with visually highlighted differences. \
+    With the `--view` argument it opens a window that lets you view the \
+    files' pages and zoom in on details.
+       
+homepage            http://vslavik.github.io/diff-pdf
+
+PortGroup           github 1.0
+github.setup        vslavik diff-pdf 0.3 v
+github.tarball_from releases
+
+PortGroup           wxWidgets 1.0
+wxWidgets.use       wxWidgets-3.0
+
+checksums           rmd160  938e1f32218a083c2db37e8c220bbde48ef091e7 \
+                    sha256  4e91f1094216b98baa22d461b9662b3b6666e49ceb01671e5db6d1a3d384c538 \
+                    size    33688
+
+depends_lib         port:poppler \
+                    port:cairo \
+                    port:wxWidgets-3.0
```

Comment:
Please sort the dependencies alphabetically.

Reply:
And please use ${wxWidgets.port} instead of the hardcoded name to make it easier to migrate to a newer version later.

Reply:
Changed to port:${wxWidgets.port}